docs/customization.md

```diff
@@ -111,9 +111,28 @@ This allows you to switch the scheme via the following javascript.
 jtd.setTheme("foo")
 ```
 
+## Override and define new variables
+
+New (v0.4.0)
+{: .label .label-green }
+
+To define new SASS variables, functions, or override existing theme variables, place SASS files in `_sass/custom/setup.scss`. This should *not* be used for defining custom styles (see the next section).
```

Comment:
I think we should write SCSS instead of SASS (or Sass) everywhere in our docs, as that's what we always use in the theme (and SCSS supports copy-paste from CSS).